.github/workflows/tck-test.yml

```diff
@@ -12,7 +12,7 @@ permissions:
 jobs:
   run_tests:
 
-    runs-on: ubuntu-20.04
```

Comment:
why this change?

Reply:
C++ test agent needs minimum version of 24 therefore I decided to just use latest as used by CI flow in up-cpp

Reply:
20.04 is older than the minimum supported version for up-cpp. The requirement has always been >=22.04.

Currently `ubuntu-latest` is an alias for `ubuntu-22.04`. It would be reasonable to lock this CI in at that version if needed (i.e. `runs-on: ubuntu-22.04`)